proposed/extended-coding-style-guide.md

```diff
@@ -420,7 +420,7 @@ Method names MUST NOT be prefixed with a single underscore to indicate
 protected or private visibility. That is, an underscore prefix explicitly has
 no meaning.
 
-Method and function names MUST NOT be declared with a space after the method name. The
+Method and function names MUST NOT be declared with space after the method name. The
```

Comment:
I'm not sure that removing the article here is correct, we need to say that we require one single space.

Reply:
That's the opposite: "MUST NOT be declared with space after the method name" but that should not matter.

Reply:
OH right! I misread.